fe/fe-core/src/main/java/com/starrocks/datacache/DataCacheSelectExecutor.java

```diff
@@ -59,7 +59,9 @@ public static DataCacheSelectMetrics cacheSelect(DataCacheSelectStatement statem
         if (connectContext.getExecutor() != null) {
             connectContext.getExecutor().registerSubStmtExecutor(stmtExecutor);
         }
+        stmtExecutor.addRunningQueryDetail(insertStmt);
         stmtExecutor.execute();
+        stmtExecutor.addFinishedQueryDetail();
```

Comment:
should we do addFinishedQueryDetail in finally?

Reply:
fixed